xbmc/filesystem/NptXbmcFile.cpp

```diff
@@ -294,16 +296,23 @@ NPT_XbmcFile::Open(NPT_File::OpenMode mode)
         }
 
         bool result;
-        CURL* url = new CURL(name);
+        CURL url(URIUtils::SubstitutePath(name));
+        CURL authUrl = url;
```

Comment:
Why the creation, then the assign to another one, when url isnt used anywhere else?

Reply:
It was needed in CFile::Stat but not here. Changed.